parts/kubernetesagentcustomdata.yml

```diff
@@ -113,7 +113,7 @@ write_files:
     DOCKER_OPTS=
     CUSTOM_CMD=/bin/true
     KUBELET_REGISTER_SCHEDULABLE=true
-    KUBELET_NODE_LABELS=role=agent
+    KUBELET_NODE_LABELS=role=agent,failure-domain.beta.kubernetes.io/region={{WrapAsVariable "location"}}
```

Comment:
What is this label used for? Why some labels are appended in kuberneteskubelet.service? I am asking because I am looking into matching node labels for Windows node with Linux node.